velox/exec/Unnest.h

```diff
@@ -39,22 +39,70 @@ class Unnest : public Operator {
   bool isFinished() override;
 
  private:
-  // Generate output for 'size' input rows starting from 'start' input row.
+  // Represents the range of rows to process and indicates that first and last
+  // row may need to be processed partially to match the output batch size.
+  // The row range is firstRowStart_ to lasRowEnd when there is
+  // single row to process, moreover, the row range for first row is
+  // firstRowStart_ to rawMaxSizes_[firstRow] and the row range for last row is
+  // 0 to lastRowEnd when there are several rows to process unless the last row
+  // process fully, in that case, use rawMaxSizes_[lastRow] as end of the last
+  // row.
```

Comment:
Comment suggestion:
> // Represents the range of rows to process and indicates that the first and last
// rows may need to be processed partially to match the configured output batch size.
// When processing a single row, the range is from 'firstRowStart_' to 'lastRowEnd'.
// For multiple rows, the range for the first row is from 'firstRowStart_' to 'rawMaxSizes_[firstRow]',
// and for the last row, it is from 0 to 'lastRowEnd', unless the last row is processed fully,
// in which case' rawMaxSizes_[lastRow]' is used as the end of the last row.